sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLoggingPolicy.java

```diff
@@ -221,11 +221,13 @@ private String prettyPrintIfNeeded(ClientLogger logger, String contentType, Stri
         return result;
     }
 
-    private long getContentLength(HttpHeaders headers) {
+    private long getContentLength(ClientLogger logger, HttpHeaders headers) {
         long contentLength = 0;
         try {
             contentLength = Long.parseLong(headers.getValue("content-length"));
         } catch (NumberFormatException | NullPointerException ignored) {
+            logger.warning("Http logging cannot parse the content-length {}."
+                + headers.getValue("content-length"));
```

Comment:
Should be passed as a parameter. And pass the 'ignored' exception as the last parameter. Though I would change its name from ignored.

```java
logger.warning("Could not parse the HTTP header content-length: '{}'.",
    headers.getValue("content-length", ignored));
```

Reply:
Done.